src/lib/updating-element.ts

```diff
@@ -317,7 +367,22 @@ export abstract class UpdatingElement extends HTMLElement {
       },
       configurable: true,
       enumerable: true
-    });
+    };
+  }
+
+  /**
+   * Returns the property options associated with the given property.
+   * These options are defined with a PropertyDeclaration via the `properties`
+   * object or the `@property` decorator and are registered in
+   * `createProperty(...)`.
+   *
+   * Note, this method should be considered "final" and not overridden. To
+   * customize the options for a given property, override `createProperty`.
+   *
```

Comment:
Maybe add a `@final` annotation. Not sure if these flow through from TypeScript to Closure via tsickle or not, but it doesn't hurt.

Reply:
Done.